src/main/java/education/cursor/movies/controller/UserController.java

```diff
@@ -28,4 +29,9 @@ public Movie getOne(@PathVariable("id") Movie movie) {
         return movie;
     }
 
-}
+    @GetMapping("category/{id}")
+    @JsonView(Views.ShortMovie.class)
+    public List<Movie> movieList(@PathVariable("id") Movie movie) {
```

Comment:
In the task, we need to search for all films by category(String value). This method searches by movie category (Category in Entity)